fabfile.py

```diff
@@ -165,6 +166,197 @@ def listdir(path):
     return run("ls " + path).split()
 
 
+@task
+def get_available_ovh_services():
+    """List all available ovh services"""
+    client = create_ovh_client()
+    result = client.get('/vps')
+    print json.dumps(result, indent=4)
+
+
+@task
+def get_domain_records(domain):
+    """Obtains all records for a specific domain"""
+    records = {}
+    client = create_ovh_client()
+    # List all Ids and get info for each one
+    record_ids = client.get('/domain/zone/{}/record/'.format(domain))
+    for record_id in record_ids:
+        info = client.get('/domain/zone/{}/record/{}'.format(domain, record_id))
+        records[info['subDomain']] = info
+    print json.dumps(records, indent=4)
+    return records
+
+
+def check_field_type_exists(field):
+    """Checks if the inputs in your dns_entries.json file are valid"""
+    type = dict(default='A', choices=['A',
+                                      'AAAA',
+                                      'CNAME',
+                                      'DKIM',
+                                      'LOC',
+                                      'MX',
+                                      'NAPTR',
+                                      'NS',
+                                      'PTR',
+                                      'SPF',
+                                      'SRV',
+                                      'SSHFP',
+                                      'TXT'])
```

Comment:
There is no need to create this dict each time you call the function. You should create this out of the function and name it other than type which override the built-in, considered bad practice.